pkg/monitoring/domainstats/prometheus/prometheus_test.go

```diff
@@ -69,7 +73,7 @@ var _ = Describe("Prometheus", func() {
 
 			ps := prometheusScraper{ch: ch}
 
-			vmStats := &stats.DomainStats{
+			vmDomainStats := &stats.DomainStats{
```

Comment:
@machadovilaca same here about a dedicated commit for refactoring

Reply:
Does this still make sense after https://github.com/kubevirt/kubevirt/pull/7814/files#r923806175 ?